src/data/accessibility-quiz.ts

```diff
@@ -1,4 +1,15 @@
 const accessibilityQuiz = [
+  {
+    Question:
+      "The correct location to identify the main language of an entire page is:",
+    Answer: "On the <html> element",
+    Distractor1: "On the <!doctype> element",
+    Distractor2: "On the <body> element",
+    Distractor3: "In a <meta> element in the <head> of the document",
+    Explanation:
+      "To set the main language of the page use the lang attribute on the <html> element at the top of the page.",
```

Comment:
```suggestion
      "To set the main language of the page, use the lang attribute on the <html> element at the top of the page.",
```

Reply:
Thank you so much for helping me out with the right English grammar and punctuation marks